packages/next/package.json

```diff
@@ -95,8 +95,8 @@
     "ws": "6.1.2"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.6.0",
```

Comment:
16.3 is supported in this change right?

Reply:
`contextType` (added in 16.6.0 https://reactjs.org/blog/2018/10/23/react-v-16-6.html#static-contexttype) is used in the `withRouter` hoc atm. Could refactor to use the `Consumer` component instead tho.

Reply:
Yeah sure please do 🙏 It makes no sense to keep the class around now that we can use a functional component 👍